### DISCUSSION TITLE
BO Product page - Unit price is not translatable

### DISCUSSION THREAD

**Discussion by florine2623** [PrestaShop/PrestaShop#35125] [Ideas]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

Same issue on Old Product Page and New Product Page.
I can't add a unit value for different language.
If I add "per kilo", il will always be "per kilo" for all the languages enables on my shop.

<img width="925" alt="Screenshot 2024-01-19 at 17 14 16" src="https://github.com/PrestaShop/PrestaShop/assets/16019289/d0f67203-39b4-4ef4-81ca-05e7d01b98f1">

### Expected behavior

I should be able to translate this element for all my languages.

### Steps to reproduce

1. Make sure you have at least 2 active languages
2. Go to BO > Products > Edit any product
3. Go to Pricing tab > Enable Display retail price per unit 
4. See that you can only add a unique value for the Unit element :x:

### PrestaShop version(s) where the bug happened

8.1.x, develop

### PHP version(s) where the bug happened

8.1

**Comment by Hlavtox** [comment]
Good find, but it's like this since ever, not really a bug. :-) We can consider it a feature request. Ping @prestashop/product-council

**Comment by AureRita** [comment]
Hi @florine2623 

Thank you for your report, as Hlavtox said, we can consider it like a feature request. 
As view with you, I'll transform this issue into discussion to speak about this new feature. 

Thank you

**Comment by RosaBenouamer** [comment]
Indeed a new feature, perhaps the same one that we have in Products>Details tab>Features. The dropdown with languages?

**Comment by Frits1969** [comment]
I do not understand. This is a mistake. There is 1 field but it is not connected to languages? When I type in English, It will read in Dutch as wel. When I chang it, it's changed in English too. But how you call it, a bug or a feature request., I like to have it fixed. How can I do that, or when is it fixed? 
Last week I told the service desk about this problem, they asked me to delete the language English in the Back Office. But I am afraid to loos a lot of text in English. So they send me to Github. Now I hope someone can help me.

**Reply by Hlavtox** [reply]
This is just a field that is not translatable by bad original design.

You can fix it on your own and contribute it to the project or hire a developer that will fix it. Do not wait for it to be fixed, it can take years.

**Comment by RosaBenouamer** [comment]
Hello @Frits1969 

Don't get us wrong, the need is legitimate, whether it's an anomaly or a new feature doesn't matter :-) We called it a new feature because it's not in the documentation, that's all. There are plenty of elements to correct or add, and we welcome any collaboration :-)
